modules/environment-roles/root.tf

```diff
@@ -106,7 +106,7 @@ resource "aws_iam_policy" "shared_terraform_policy_3" {
 }
 
 resource "aws_iam_policy" "shared_terraform_policy_4" {
-  policy = templatefile("${path.module}/templates/shared_terraform_policy_4.json.tpl", { environment = title(var.tdr_environment), account_id = data.aws_caller_identity.current.account_id, sub_domain = var.sub_domain })
+  policy = templatefile("${path.module}/templates/shared_terraform_policy_4.json.tpl", { environment = title(var.tdr_environment), account_id = data.aws_caller_identity.current.account_id, sub_domain = var.sub_domain, environment_lower = var.tdr_environment })
```

Comment:
Is there a reason why the template variable is named `environment_lower`?

Reply:
There's already an template variable there called environment which is title case but I needed all lower case environment. I'm happy to change it though.

Reply:
Ah, maybe change it to `environment_lower_case` just to make it a bit clearer. I thought it meant a "lower" environment.

Reply:
Yeah makes sense, I've changed it.